{"name": "test_suite_login", "status": "failed", "statusDetails": {"message": "AssertionError: Test case 2 failed\nassert False\n +  where False = correct_credential_login(<selenium.webdriver.chrome.webdriver.WebDriver (session=\"ae99ad11d28302f0d92d5842f42a6628\")>)", "trace": "headless_driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ae99ad11d28302f0d92d5842f42a6628\")>\n\n    def test_suite_login(headless_driver):\n        #browser driver\n        driver = headless_driver\n    \n        #test case 1\n        assert incorrect_password_login(driver), \"Test case 1 failed\"\n    \n        #test case 2\n>       assert correct_credential_login(driver), \"Test case 2 failed\"\nE       AssertionError: Test case 2 failed\nE       assert False\nE        +  where False = correct_credential_login(<selenium.webdriver.chrome.webdriver.WebDriver (session=\"ae99ad11d28302f0d92d5842f42a6628\")>)\n\ntests\\test_suites\\test_suite_login.py:30: AssertionError"}, "start": 1735203727235, "stop": 1735203734039, "uuid": "3b12e3a0-4b82-427c-81e6-35d4d99778fd", "historyId": "1bed669dc312d0c46ce9f96d3ecfbe0c", "testCaseId": "1bed669dc312d0c46ce9f96d3ecfbe0c", "fullName": "tests.test_suites.test_suite_login#test_suite_login", "labels": [{"name": "parentSuite", "value": "tests.test_suites"}, {"name": "suite", "value": "test_suite_login"}, {"name": "host", "value": "LOQ15"}, {"name": "thread", "value": "20956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_suites.test_suite_login"}]}